### PR TITLE
Update README for readability and emoji use!

### PR DESCRIPTION
**Pull request by monalisaoctocat**
✨ Isn't this better 🎉  ✨

### CODE REVIEW

**Comment by tarebyte** [comment]
Looks great!

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# effective-enigma
\ No newline at end of file
+# Effective Enigma
+
+This is our new awesome repository!
+
+:smiley:
```
Comment (monalisaoctocat) [inline README.md]: Needs more emoji!

Comment (antonyggvzvmnxxcx) [reply README.md]: eca6c09c69177181714c9c5fa5c9b6775efbdf2e

**Comment by antonyggvzvmnxxcx** [comment]
> ✨ Isn't this better 🎉  ✨

**Pull request merged**